isst/README.md

```diff
@@ -0,0 +1,32 @@
+# Release Notes for Intel® Speed Select Technology test cases
+
+Intel® Speed Select Technology: a collection of features improve performance 
+and optimizes the total cost of ownership (TCO) by proving more control 
+over the CPU performance.
+
+The isst cases are designed for Intel® Architecture-based server platforms,
+supported beginning from Cascade lake, then Cooper lake, Icelake and future platforms
+
+The prerequisites to run the ISST cases:
+## The server CPU supports the ISST feature 
+## BIOS enabled Dynamic SST-PP setting
+## User needs intel-speed-select, turbostat, stress tools
```

Comment:
Better to use **list** syntax here.

- The server CPU supports the ISST feature 
- BIOS enabled Dynamic SST-PP setting
- User needs intel-speed-select, turbostat, stress tools

Reply:
Will change, thanks!